src/app/services/pagination.service.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable } from '@angular/core';
+
+@Injectable()
+
+export class PaginationService {
+
+    getTotalPages(recipesNumber: number): number {
```

Comment:
service should be universal, that's why here should be not recipesNumber but itemsNumber